.github/workflows/integrations-test.yml

```diff
@@ -0,0 +1,53 @@
+name: Integration Tests
+
+# Only run workflow manually
+# Refer to https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch
+on:
```

Comment:
This file need to be on master branch before, we can manually invoke specific workflows